src/kt_class.cpp

```diff
@@ -7,10 +7,10 @@ KtClass::KtClass(jni::JObject p_wrapped, jni::JObject& p_class_loader) : JavaIns
 }
 
 KtObject* KtClass::create_instance(jni::Env& env, const Variant** p_args, int p_arg_count, Object* p_owner) {
-    auto new_method = get_method_id(env, "new", "(J[Lgodot/core/KtVariant;)Lgodot/core/KtObject;");
+    jni::MethodId new_method = get_method_id(env, "new", "(J[Lgodot/core/KtVariant;)Lgodot/core/KtObject;");
     // TODO: send args
-    auto j_kt_object = wrapped.call_object_method(env, new_method, {reinterpret_cast<long>(p_owner)});
-    return new KtObject(j_kt_object, class_loader);;
+    const jni::JObject &j_kt_object = wrapped.call_object_method(env, new_method, {reinterpret_cast<long>(p_owner)});
```

Comment:
the returned object is not a reference nor is const: so it should be `jni::JObject j_kt_object`.